base-system/src/main/java/org/ibankapp/base/system/NetCard.java

```diff
@@ -0,0 +1,58 @@
+package org.ibankapp.base.system;
```

Comment:
### 版权声明模板
    /*
     * iBankApp
     *
     * License : Apache License,Version 2.0, January 2004
     *
     * See the LICENSE file in English or LICENSE.zh_CN in chinese
     * in the root directory or <http://www.apache.org/licenses/>.
     */

可以将以上内容加到idea的配置中，如下图

![2017-06-09 9 48 03](https://user-images.githubusercontent.com/3068575/26957998-eee7ba48-4cfa-11e7-9975-d4795cb0d448.png)

### class JavaDoc举例

    /**
     * 异常处理基类
     *
     * @author <a href="http://www.ibankapp.org">ibankapp</a>
     * @author <a href="mailto:codelder@ibankapp.org">codelder</a>
     * @since 1.0.0.0
     */